### PR TITLE
[docs]: Fix documentation for the Circular Progressbar Component

### LINKED ISSUE #11037: [docs]: Update the documentation for the Circular Progressbar Component
### The entire URL of the documentation with the issue

https://docs.tooljet.com/docs/widgets/circular-progress-bar

**Formatting updates:**
- Replace  "`Fx`" and "Fx" with "**fx**" wherever applicable. ( i.e. remove the backticks (``) make it lowercase and bold -> **fx**)
- For each h2 (created using ##), remove 24px padding-bottom. The `div` that wraps up the h2 headers and the section below the header should only have a "padding-top" of 24px and no "padding-bottom". 

**Content updates:**
- Replace the word "widget" with "component" wherever applicable. ( Please do not update the URLs )
- Layout Tabel
  - Update the title name from "Layout" to "Devices" [reference](https://docs.tooljet.com/docs/widgets/text-input/#devices)
  - Add the "Expected value" column and update the content in the "Expected value" and "Description" columns as per the given [reference](https://docs.tooljet.com/docs/widgets/text-input/#devices) ( Please just check the sentence structure from the given reference )
- Add a divider before the "Styles" section [reference](https://docs.tooljet.com/docs/widgets/textarea#styles)

**The changes are to be implemented in two files -** 
 Next - ToolJet/docs/docs/widgets/circular-progress-bar.md
 Version 2.50.0 (LTS) - ToolJet/docs/versioned_docs/version-2.50.0-LTS/widgets/circular-progress-bar.md

### If the issue is confirmed, would you be willing to submit a pull request?

No

### Code of Conduct

- [X] I agree to follow the ToolJet Code of Conduct

### PR DESCRIPTION
**Pull request by DeepanshuYadav-code**
will close #11037

### CODE REVIEW

File: docs/docs/widgets/circular-progressbar.md [hunk 1]
```diff
@@ -6,60 +6,60 @@ title: Circular Progressbar
...
-| Show on mobile | Toggle on or off to display the widget in mobile view. You can programmatically determine the value by clicking on Fx to set the value `{{true}}` or `{{false}}`. |
+| <div style={{ width:"100px"}}> Property </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"100px"}}> Expected Value </div> |
+| ----------- | ----------- | ----------- |
+| Show on desktop | Makes the component visible in the desktop view. | You can set it with the toggle button or dynamically configure the value by clicking on **fx** and entering a logical expression. |
+| Show on mobile | Makes the component visible in mobile view. | You can set it with the toggle button or dynamically configure the value by clicking on **fx** and entering a logical expression. |
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
```
Comment (RudraPratik30) [inline docs/docs/widgets/circular-progressbar.md:63]: Hey @DeepanshuYadav-code you need to add a divider before the style section.

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]
Hey @DeepanshuYadav-codes please make the mentioned changes in both versions, thanks!

**Comment by DeepanshuYadav-code** [comment]
hey @RudraPratik30, thanks for the timely review. I have add the divider before the styles section.

**Review by RudraPratik30 — APPROVED** [review]
Hey @DeepanshuYadav-code, thanks for contributing!
The changes look good to me and forwarding it to the team for final review.
You can expect your PR to be merged soon.

**Review by karan-rathod-316 — APPROVED** [review]
Looks good to me.

**Pull request merged**